bin/install.sh

```diff
@@ -32,9 +32,16 @@ if [ -z "${JULIA_VERSION}" ]; then
     stop "JULIA_VERSION is not defined; don't know what to download"
 fi
 
-MAJOR="$(echo "${JULIA_VERSION}" | cut -d'.' -f 1)"
-MINOR="$(echo "${JULIA_VERSION}" | cut -d'.' -f 2)"
-PATCH="$(echo "${JULIA_VERSION}" | cut -d'.' -f 3)"
+if [[ $JULIA_VERSION == *.* ]]; then
```

Comment:
Note that this script is run as POSIX sh (`/bin/sh`), but the syntax used here is specific to bash. I'm not actually sure how this worked on CI, since bash is not installed by default on FreeBSD.

Reply:
I did not find an easy way to make the same check with `sh`, but `cut` has a `-s` to not print lines without the delimiter, which applies to this case, so I used it.

Reply:
Perfect 👍